### ISSUE TITLE
Invalide code generation when using "option deprecated = true;"

### ISSUE THREAD

**Issue by ligi** [square/wire#1468]
I face a problem with when using protobuf files that contain "option deprecated = true;"
if so the generated code contains the usage and import of "com.google.protobuf.MessageOptions" that cannot be resolved.

**Comment by oldergod** [comment]
Would you be able to write a test for this?

**Comment by ligi** [comment]
I wrote a minimal demo project that shows the problem:
https://github.com/ligi/wire_bug_deprecation
It fails with:

```
./gradlew clean build                                                                                                                       

> Task :failing:generateProtos
Writing wire.bug.demo.Success to /home/ligi/git/wire_bug_deprecation/failing/build/generated/source/wire

> Task :failing:compileJava FAILED
/home/ligi/git/wire_bug_deprecation/failing/build/generated/source/wire/wire/bug/demo/Success.java:5: error: package com.google.protobuf does not exist
import com.google.protobuf.MessageOptions;
                          ^
/home/ligi/git/wire_bug_deprecation/failing/build/generated/source/wire/wire/bug/demo/Success.java:25: error: cannot find symbol
  public static final MessageOptions MESSAGE_OPTIONS = new MessageOptions.Builder()
                      ^
  symbol:   class MessageOptions
  location: class Success
/home/ligi/git/wire_bug_deprecation/failing/build/generated/source/wire/wire/bug/demo/Success.java:25: error: package MessageOptions does not exist
  public static final MessageOptions MESSAGE_OPTIONS = new MessageOptions.Builder()
                                                                         ^
3 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':failing:compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 2s
4 actionable tasks: 4 executed

```

as soon as you comment out the line  "option deprecated = true;" [here](https://github.com/ligi/wire_bug_deprecation/blob/master/failing/src/main/proto/messages.proto#L5) it will compile fine

**Comment by oldergod** [comment]
Thank you. This is fixed in `3.2.2`.
